SyncIntegrationTests/xcodebuild.py

```diff
@@ -10,7 +10,7 @@
 
 class XCodeBuild(object):
     binary = 'xcodebuild'
-    destination = 'platform=iOS Simulator,name=iPhone 11,OS=13.2.2'
+    destination = 'platform=iOS Simulator,name=iPhone 11'
```

Comment:
The available `OS` values on my machine are different, perhaps due to newer OS or XCode version. Leaving out the `OS` part defaults it to the latest available OS, which I think should be OK?

Reply:
That part was added due to a WebView issue apple had on iOS 13.3, it was not possible to interact with the fields in the FxA sign in page so no way to enter Email/Password or tap on Continue or Sing in buttons. I workaround that by specifying the latest version in which that worked. 
Fortunately this has been fixed in 13.4 which we are already using, so yeah, that  can be removed, it was in fact in my TODO list ;) 
Thanks!